### DISCUSSION TITLE
How can i filter what to count in _count

### DISCUSSION THREAD

**Discussion by ADrejta** [prisma/prisma#9107] [Q&A] [answered]
Hi everyone.

I'm trying to filter what to count in the _count part of counting relations.

Let's take the example of users and posts:
```js
 await prisma.user.findMany({
  include: {
    _count: {
      select: { posts: true },
    },
  },
})
```
Let's say that posts have a property 'published' that can be true or false.
How can i count how many posts, that have the published property equal to true, each user has (including returning users with no published posts with a value of 0)?

It doesn't seem like i can do a where clause inside the _count block.

**Comment by ryands17** [comment, marked as answer]
@ADrejta 👋 

Unfortunately this isn't supported yet. It would be great if you could add a 👍 to this request so that we know the priority.
https://github.com/prisma/prisma/issues/8413

As a workaround, you would need to use a raw query.

**Reply by ADrejta** [reply]
Should i create an issue from this discussion? Or just leave it here?
P.s
Sorry didn't see the issue number

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
